packages/react-native-web/src/exports/InteractionManager/__tests__/index-test.js

```diff
@@ -0,0 +1,323 @@
+/**
+ * Copyright (c) Nicolas Gallagher.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+function expectToBeCalledOnce(fn) {
+  expect(fn.mock.calls.length).toBe(1);
+}
+
+describe('InteractionManager', () => {
+  let InteractionManager;
+  let interactionStart;
+  let interactionComplete;
+
+  beforeEach(() => {
+    jest.resetModules();
+    InteractionManager = require('..');
```

Comment:
Why is InteractionManager being required before each test?

Reply:
`InteractionManager` has local variables and queue. This can easily avoid conflicts between these test cases. https://jestjs.io/docs/28.x/jest-object#jestresetmodules.